winml/lib/Api.Ort/OnnxruntimeDescriptorConverter.cpp

```diff
@@ -412,7 +416,7 @@ CreateTensorFeatureDescriptor(
 
   auto shape = std::vector<int64_t>(num_dims);
   if (auto status = engine_factory->UseOrtApi()->GetDimensions(tensor_info, shape.data(), shape.size())) {
-      throw;  //TODO fix throw here!;
+    throw;  //TODO fix throw here!;
```

Comment:
TODO

Reply:
same above

---
In reply to: [368191885](https://github.com/microsoft/onnxruntime/pull/2869#discussion_r368191885) [](ancestors = 368191885)